spmd/tensor/api.py

```diff
@@ -212,6 +222,15 @@ def __torch_function__(cls, func, types, args=(), kwargs={}):
     # pyre-fixme[3]: Return type must be annotated.
     # pyre-fixme[2]: Parameter must be annotated.
     def __torch_dispatch__(cls, func, types, args=(), kwargs=None):
+
```

Comment:
I did this because otherwise I'd have to deal directly with the _reshape_alias op which requires playing with strides. Basically if I only implement @register_sharding_prop then i'm forced to re-compute local strides based on global strides.